manifests/m/michidk/vscli/0.1.4/michidk.vscli.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: michidk.vscli
+PackageVersion: 0.1.4
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: vscli-x86_64-pc-windows-msvc/vscli.exe
+  PortableCommandAlias: vscli
+ReleaseDate: 2023-06-17
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/michidk/vscli/releases/download/v0.1.4/vscli-x86_64-pc-windows-msvc.zip
+  InstallerSha256: 686E7BA00FC75C78A8E38459357D0F4F99703552DF82F9B424706F0F1A987137
+  Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64

```

```suggestion
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
YAML uses indenting to specify data properties. Having these at the same indent made them equal under this property of Installers, instead of making PackageDependencies a property of Dependencies.